conf/devel.yaml

```diff
@@ -1,13 +1,91 @@
 ---
-stenographer_instances:
-    - { host: 127.0.0.1, sensor: sensor-001, port: 1234, key: /etc/stenographer/certs/client_key.pem, cert: /etc/stenographer/certs/client_cert.pem, ca: /etc/stenographer/certs/ca_cert.pem }
+# Configure your stenographer instances here. sensor is a just an arbitrary unique name
+STENOGRAPHER_INSTANCES:
+    - { host: 127.0.0.1, sensor: sensor-1, port: 1234, key: /etc/stenographer/certs/client_key.pem, cert: /etc/stenographer/certs/client_cert.pem, ca: /etc/stenographer/certs/ca_cert.pem }
 
 # Flask config values
 DEBUG: true
 TESTING: true
-SECRET_KEY: CHANGE_THIS_SECRET_KEY_TO_RANDOM_VALUE
+SECRET_KEY: CHANGE_THIS_SECRET_KEY_TO_ANY_RANDOM_VALUE
 SESSION_COOKIE_NAME: DOCKET_SESSION
 USE_X_SENDFILE: false
-LOGGER_NAME: docket
+
+# Celery configuration - CELERY_URL is many things :/
 CELERY_URL: redis://localhost:6379
+# all results are simply written to disk, at the moment there's no need to care about brokering them
+CELERY_TASK_IGNORE_RESULT: true
+
+# Docket configuration
+# SPOOL_DIR     - where captures are saved and served - if you change this make sure to fix the webserver's static file configuration
 SPOOL_DIR: /var/spool/docket/
+# WEB_ROOT      - the base url for all docket requests: 
+# WEB_ROOT=/foo/ => http://HOST:PORT/foo/status, http://HOST:PORT/foo/q/after/5m/host/1.2.3.4
+WEB_ROOT: /
+# DATE_FORMAT   - strftime format for showing a datetime to a user, not used yet
+DATE_FORMAT: '%Y%jT%H:%M:%S'
+# Logging configuration
+LOGGER_NAME: docket
+# LOG_LEVEL  (error, warning, info, debug) any other value will be ignored
+LOG_LEVEL: debug
+LOG_MSG_FORMAT: "[%(processName)-8s:%(thread)d] %(message)s"
+LOG_DATE_FORMAT: '%Y-%m-%dT%H:%M:%SZ'
+LOG_DATE_UTC: True
+# LOG_FILE   a secondary logging handler - this file will be rotated at midnight.
+LOG_FILE: /var/log/docket/docket.log
+LOG_FILE_MSG_FORMAT: "%(asctime)s[%(processName)-8s:%(thread)d] %(message)s"
+#LOG_DATE_FORMAT: '%Y-%m-%dT%H:%M:%SZ'  # ISO
+LOG_FILE_DATE_FORMAT: '%Y%jT%H:%M:%S'
```

Comment:
Same as previous comment on RFC3339

Reply:
Same answer - this value is just the date format for LOG_FILE output.